README.md

```diff
@@ -24,6 +24,8 @@
 
 This component creates a connection to an SFTP server to read and upload files.
 
+The optional environment variable `MAX_FILE_SIZE` should be set in settings to provide the maximum file size that can be uploaded in **bytes**. The default value for `MAX_FILE_SIZE` is 3.5GB.
```

Comment:
Max size should be 100 MB.  It's not clear if that is `100*1000^2` or `100*1024^2` bytes.

Reply:
It is the latter; file sizes are returned in bytes